Makefile

```diff
@@ -46,9 +46,9 @@ TOOLCHAIN_BIN = $(TOOLCHAIN_DIR)/bin
 TOOLCHAIN_GOLANG_DIR = $(TOOLCHAIN_DIR)/golang
 PROTOC := $(TOOLCHAIN_BIN)/protoc
 PROTOC_INCLUDES := $(TOOLCHAIN_DIR)/include/
-GCP_PROJECT_ID = "Set $GCP_PROJECT_ID in your bashrc."
+GCP_PROJECT_ID ?= open-match-public-images
```

Comment:
Please revert this line. This will cause privileged devs to accidentally overrwrite the public images during testing.